<?xml version="1.0"?>
<launch>
	<arg name="TestDevice" value="Spacenav"/>
	<include file="$(find masterslave)/launch/BasicUrsula.launch"/>
	<node pkg="masterslave" type="NumericKinematicCommander" name="NumericKinematicCommander" args="$(arg TestDevice)" output="screen"/>
	<node pkg="masterslave" type="NumericKinematic" name="NumericKinematic" output="screen"/>
	<node pkg="masterslave" type="RosOpenIGTLBridge" name="RosOpenIGTLBridge" output="screen"/>




</launch>

